dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -0,0 +1,79 @@
+[[expanding_persistent_volumes]]
+= Expanding Persistent Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[overview]]
+== Overview
+
+You can increase the size of the persistent volumes (PVs) that your pods are using. To do this, you can update the persistent volume claim (PVC) to request a new size. The  controller then applies the change to the PV, which is bound to the PVC.
+
```

Comment:
Then, the controller applies
(also there may be an extra space between "the" and "controller")

Reply:
I wouldn't mention any of the 'internal workings' of this, it will confuse the user..  (no controller, no attributes on PVC objects, nothing on how it works.. this should only show the UX..